libcudacxx/include/cuda/std/__type_traits/is_member_function_pointer.h

```diff
@@ -27,20 +27,6 @@
 
 _LIBCUDACXX_BEGIN_NAMESPACE_STD
 
-#if defined(_LIBCUDACXX_IS_MEMBER_FUNCTION_POINTER) && !defined(_LIBCUDACXX_USE_IS_MEMBER_FUNCTION_POINTER_FALLBACK)
-
-template <class _Tp>
-struct _CCCL_TYPE_VISIBILITY_DEFAULT is_member_function_pointer
-    : public integral_constant<bool, _LIBCUDACXX_IS_MEMBER_FUNCTION_POINTER(_Tp)>
-{};
-
-#  if _CCCL_STD_VER > 2011 && !defined(_LIBCUDACXX_HAS_NO_VARIABLE_TEMPLATES)
-template <class _Tp>
-_LIBCUDACXX_INLINE_VAR constexpr bool is_member_function_pointer_v = _LIBCUDACXX_IS_MEMBER_FUNCTION_POINTER(_Tp);
-#  endif
-
-#else
-
```

Comment:
why is `__libcpp_is_member_pointer` defined unconditionally now?

Reply:
That is just github formatting being bad.

We need the `__libcpp_is_member_pointer` in other places, so I moved that to the top